zips/zip-0227.rst

```diff
@@ -337,43 +335,98 @@ For the ``IssueBundle``:
 
 **Note:** that the commitment is not included in the ``IssuanceAction`` itself. As explained below, it is computed later by the validators and added to the note commitment tree.
 
+Specification: Reference Notes and Global Issuance State
+========================================================
 
-Specification: Consensus Rule Changes
-=====================================
+Reference Notes
+---------------
 
-For the ``IssueBundle``:
+A reference note for a Custom Asset MUST be included by the issuer as the first Note in the Action of the Issuance Bundle where that Custom Asset is being issued for the first time.
+
+A reference note for a Custom Asset is an Issue Note where the value $\mathsf{v}$ is set to $0$, the Asset Base ($\mathsf{AssetBase}$) corresponds to that of the Custom Asset, and the recipient address $(\mathsf{d}, \mathsf{pk}_{\mathsf{d}})$ is set to the default diversified payment address (i.e. the diversified payment address with diversifier index $0$) derived from the all-zero Orchard spending key using the algorithm specified in §4.2.3 of the protocol specification [#protocol-orchardkeycomponents]_. This corresponds to a 43 byte ``u8`` array with the following entries::
```

Comment:
Confirming that the all-zeroes spending key doesn't return an error from the `orchard` crate's key derivation logic? It's almost certainly the case given the presence of the encoded address below, and this should be a cryptographically negligible case.

Reply:
Yes, we have a test checking that the value mentioned in the ZIP equals that derived as specified in the spec. The link to that test is [here](https://github.com/QED-it/orchard/blob/01e85a5f116bdf50c78ded08a820d853c33d0bd8/src/constants/reference_keys.rs#L47).